src/components/layers/basemaps/BasemapCard.js

```diff
@@ -130,6 +118,20 @@ const BasemapCard = props => {
                             opacity={opacity}
                             onChange={opacity => changeBasemapOpacity(opacity)}
                         />
+                        <div>
```

Comment:
@turban I added this empty `<div>` wrapper because `<CardActions>` from material-ui assigns a `className` property to all children, but `<Tooltip>` throws doesn't support `className` and so throws an error when it is a first-level child of `<CardActions>`.  Let me know if you know of a better way to work around this.

Reply:
Actually, I just tested `React.Fragment` and it works, that will be better since it won't pollute the DOM.  I had previously tried the shorthand `<>` but it didn't work with our Webpack configuration.

Reply:
Updating the PR now